packages/flame/lib/src/experimental/world.dart

```diff
@@ -20,4 +21,13 @@ class World extends Component {
     assert(CameraComponent.currentCamera != null);
     super.renderTree(canvas);
   }
+
+  @override
+  Iterable<ComponentPoint> componentsAtPoint(Vector2 point) {
+    return const Iterable.empty();
+  }
+
+  Iterable<ComponentPoint> componentsAtPointFromCamera(Vector2 point) {
```

Comment:
What does the `FromCamera` part mean here?

Reply:
It means it's an internal function that's supposed to be accessed by `Camera` only -- similar to `renderFromCamera()`. This is needed because only the camera knows how to construct the `point` in the world's local coordinate space.

Reply:
Alright, should it be marked as `@internal` then?

Reply:
Yep, just did.